lib/Indexer.php

```diff
@@ -49,7 +48,7 @@ public function updateSearchIndex(array $contentIds, $commit)
         foreach ($contentIds as $contentId) {
             try {
                 $info = $contentHandler->loadContentInfo($contentId);
-                if ($info->isPublished) {
+                if ($info->status === ContentInfo::STATUS_PUBLISHED) {
```

Comment:
There's no `isPublished` property, but there is a method that does exactly same comparison.

```suggestion
                if ($info->isPublished()) {
```